pkg/virt-config/feature-gates.go

```diff
@@ -172,3 +163,50 @@ func (config *ClusterConfig) AlignCPUsEnabled() bool {
 func (config *ClusterConfig) NodeRestrictionEnabled() bool {
 	return config.isFeatureGateEnabled(featuregate.NodeRestrictionGate)
 }
+
+func GetEnabledFeatureGates(kubevirtDevConfig *v1.DeveloperConfiguration) []string {
+	if kubevirtDevConfig == nil || (kubevirtDevConfig.FeatureGates == nil && kubevirtDevConfig.FeatureGatesMap == nil) {
+		return nil
+	}
+
+	featureGateMap := maps.Clone(kubevirtDevConfig.FeatureGatesMap)
+	if featureGateMap == nil {
+		return slices.Clone(kubevirtDevConfig.FeatureGates)
+	}
+
+	for _, fgFromList := range kubevirtDevConfig.FeatureGates {
+		if _, existsInMap := featureGateMap[fgFromList]; !existsInMap {
+			featureGateMap[fgFromList] = true
+		}
+	}
+
+	enabledFeatureGates := make([]string, 0, len(featureGateMap))
+	for fg, enabled := range featureGateMap {
+		if enabled {
+			enabledFeatureGates = append(enabledFeatureGates, fg)
+		}
+	}
+
+	return enabledFeatureGates
+}
+
+func IsFeatureGateEnabled(featureGate string, kubevirtDevConfig *v1.DeveloperConfiguration) bool {
+	if kubevirtDevConfig == nil {
+		return false
+	}
+
+	if fg := featuregate.FeatureGateInfo(featureGate); fg != nil && fg.State == featuregate.GA {
+		return true
+	}
```

Comment:
I think it should be a feature of the featureGate type; e.g.

```go
func (fg *FeatureGate) IsGAed() bool {
	return fg != nil && fg.State == GA
}
```

Then use it here:
```suggestion
	if featuregate.FeatureGateInfo(featureGate).isGAed() {
		return true
	}
```

Reply:
Done!
Although I had to check if `fg == nil`, so it's not much simplified, but still I like the idea :)

Reply:
Done! :)
Thank you @nunnatsa!